drafts/WEP0001.rst

```diff
@@ -0,0 +1,273 @@
+:WEP-Number: 0001
+:Created: 2020-01-13
+:Last-Modified: 2020-01-13
+:Author: Sasha Romijn
+:Status: Draft 
+
+WEP1 - Opportunity Grant policy for Write the Docs conferences
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Abstract
+========
+
+This WEP proposes a policy for offering opportunity grants (known as
+financial aid in some other events) for Write the Docs conferences.
+
+Motivation
+==========
+
+Write the Docs conferences sometimes support speakers and attendees
+with their travel and accommodation costs. There is currently no
+structural policy for this, and this WEP intends to define a clear
+process for offering and processing grants.
+
+Proposal
+========
+
+Overview
+--------
+
+Write the Docs conferences can offer grants to attendees and speakers.
+The possibilities depend on the available budget, which may be different
+per conference.
+
+Some important considerations apply to grants:
+
+* There are generally more applications than funds available, so there must
+  be a process to prioritise grant allocation.
+* People can be reluctant, or even feel shameful about applying, so the process
+  for applications should be encouraging and welcoming, even for people who
+  feel they might not deserve it.
+* Grants improve access for marginalised and underrepresented groups already,
+  but an option can be to prioritise marginalised groups in allocations.
+  Currently, this proposal is written with prioritisation of marginalised and
+  underrepresented groups.
+
+Speakers
+--------
+
+Accepted speakers are an essential part of the conference, and should
+therefore have a special grant priority. All speakers and organisers will
+receive a free ticket, as they already pay for their participation in the
+conference with their volunteer time.
+
+Either at the time of the CFP (in which case it should be stressed on this
+form that choices will not influence talk selection) or later, all speakers
+will be asked whether (pick one option):
+
+* They are willing and able to pay their own costs completely.
+* Their employer is funding their travel and accommodation.
+* They would like to request a (partial) grant (if so, we need their details -
+  a partial grant could mean "I need you to cover my flight, but I can arrange
+  my own accommodation".)
+
+If their employer is funding their attendance, the employer will be recognised
+as a low level sponsor (perhaps a bit higher in case of very expensive travel).
```

Comment:
This adds a reasonably amount of overhead, but I generally agree it's a good idea. I imagine @plaindocs has more insight here, but I don't know a great way to handle this without making it a bit awkward and work for the speaker (eg. rounding up marketing folks, getting a logo approved, etc.)

Reply:
The minimum we can do is make this at the speaker's discretion. It's supposed to be more of a benefit to the speaker, not supposed to be a burden.

Reply:
Unless speakers are struggling to convince employers to send them, it is a benefit to employers not to speakers themselves. So I'd tend to suggest a low effort alternative of listing employers who send speakers in text, without logo. :-p Win-win-win?

Reply:
I see you've addressed this in the next paragraph, sorry. Even so, would it make sense to add this if we find out that speakers are struggling to get support from employers?

Reply:
I am ambivalent about this. It's been a long time since I've been an employee. I am fine with dropping it for now and considering it for later.

Reply:
It has been dropped from the text.